chart/values.yaml

```diff
@@ -0,0 +1,3071 @@
+##
+commonConfiguration: |
+  # CARTO Self hosted configuration
+
+  # Version
+  CARTO_ONPREMISE_VERSION="{{ .Chart.AppVersion }}"
+
+  # CARTO Self hosted configuration
+  # >>>>>>>>>>>>  TO BE CHANGED  >>>>>>>>>>>>>>>>>>>>>>>>>
+
+  # Fully qualified domain where CARTO will be accesible
+  ONPREM_DOMAIN="{{ include "carto.baseUrl" . }}"
+
+  # If you want to use Google Maps and have a API KEY set it here
+  # REACT_APP_GOOGLE_MAPS_API_KEY="CUSTOMIZED_FOR_USER_IF_NEEDED"
+
+  # Configuration onfiguration for PostgreSQL
+  WORKSPACE_POSTGRES_HOST="{{ include "carto.postgresql.host" . }}"
+  WORKSPACE_POSTGRES_PORT="{{ include "carto.postgresql.port" . }}"
+
+  # Customer project and tenant name
+  ONPREM_TENANT_ID="onp-<FILL_ME>"
+  ONPREM_GCP_PROJECT_ID="carto-tnt-${ONPREM_TENANT_ID}"
+
+  # Change the following and read the documentation to use your own buckets
+  WORKSPACE_GCS_THUMBNAILS_BUCKET="${ONPREM_GCP_PROJECT_ID}-thumbnails-storage"
+  WORKSPACE_GCS_DATASETS_BUCKET="${ONPREM_GCP_PROJECT_ID}-client-storage"
+  IMPORT_GCS_DATA_IMPORTS_BUCKET="${ONPREM_GCP_PROJECT_ID}-import-storage"
+
+  # Your custom certificates, read the documentation
+  # ROUTER_SSL_AUTOGENERATE="<1 to enable | 0 to disable>"
+  # ROUTER_SSL_CERTIFICATE_PATH="/etc/nginx/ssl/<cert>.crt"
+  # ROUTER_SSL_CERTIFICATE_KEY_PATH="/etc/nginx/ssl/<cert>.key"
+
+  # >>>>>>>>>>>>  END OF CHANGES  >>>>>>>>>>>>>>>>>>>>>>>>>
+
+  CARTO_AUTH0_CLIENT_ID="<FILL_ME>"
+  CARTO_AUTH0_CUSTOM_DOMAIN="<FILL_ME>"
+
+  ACC_DOMAIN="<FILL_ME>"
+  ACC_GCP_PROJECT_ID="<FILL_ME>"
+  ACC_GCP_PROJECT_REGION="<FILL_ME>"
+
+  CARTO_ONPREMISE_CARTO_DW_LOCATION="<FILL_ME>"
+
+  # <<<<<<<<<<<<<  END OF CUSTOMER CONFIG <<<<<<<<<<<<<<<<<<<
+
+  ### DO NOT EDIT BELOW THIS LINE
+
+  GOOGLE_APPLICATION_CREDENTIALS="../certs/key.json"
+  CARTO3_ONPREM_VOLUMES_BASE_PATH="./"
+
+  DOCKER_REGISTRY_BASE_PATH="gcr.io/carto-onprem-artifacts"
+  CARTO_ONPREMISE_AUTH0_CLIENT_ID="${CARTO_AUTH0_CLIENT_ID}"
+  WORKSPACE_THUMBNAILS_BUCKET="${ONPREM_GCP_PROJECT_ID}-thumbnails-storage"
+  WORKSPACE_DATASETS_BUCKET="${ONPREM_GCP_PROJECT_ID}-client-storage"
+  ### END VALUES FOR EACH ONPREM #########################
+
+  ### GENERIC CONF
+
+  ### WWW variables
+  REACT_APP_CLIENT_ID="${CARTO_AUTH0_CLIENT_ID}"
+  REACT_APP_AUTH0_DOMAIN="${CARTO_AUTH0_CUSTOM_DOMAIN}"
+  REACT_APP_ACCOUNTS_API_URL="https://${ACC_DOMAIN}"
+  REACT_APP_ACCOUNTS_URL="https://${ONPREM_DOMAIN}/acc/"
+  REACT_APP_WORKSPACE_API_URL="https://${ONPREM_DOMAIN}/workspace-api"
+  REACT_APP_API_BASE_URL="https://${ONPREM_DOMAIN}/api"
+  REACT_APP_PUBLIC_MAP_URL="https://${ONPREM_DOMAIN}/workspace-api/maps/public"
+  REACT_APP_AUTH0_AUDIENCE="carto-cloud-native-api"
+  REACT_APP_WORKSPACE_URL_TEMPLATE="https://{tenantDomain}"
+  REACT_APP_CUSTOM_TENANT="${ONPREM_TENANT_ID}"
+  REACT_APP_IMPORT_DATASET="carto_dw.carto-dw-{account-id}.shared"
+  REACT_APP_HUBSPOT_ID="474999"
+  REACT_APP_HUBSPOT_LIMIT_FORM_ID="cd9486fa-5766-4bac-81b9-d8c6cd029b3b"
```

Comment:
Do we want to hardcode this values here?

Reply:
This comes from the settings the CARTO team gives to their customers. They should be the ones deciding on this.